ctapipe/core/tool.py

```diff
@@ -119,16 +134,18 @@ def main():
 
     """
 
-    config_file = Path(
-        exists=True,
-        directory_ok=False,
-        allow_none=True,
-        default_value=None,
-        help=(
-            "name of a configuration file with "
-            "parameters to load in addition to "
-            "command-line parameters"
-        ),
+    config_file = List(
+        trait=Path(
+            exists=True,
+            directory_ok=False,
+            allow_none=True,
+            default_value=None,
+            help=(
+                "name of a configuration file with "
```

Comment:
Expand `help` to explain we can have multiple config files and how precedence works?